README.md

```diff
@@ -107,14 +114,27 @@ dataloader = torch.utils.data.DataLoader(
     shuffle=True,           # shuffling is important!
     collate_fn=collate_fn)  # apply transformations to the input images
 
+
+# create a PyTorch module for the SimCLR model
+class SimCLR(nn.Module):
+    def __init__(self, backbone):
+        super().__init__()
+        self.backbone = backbone
+        self.projection_head = models.modules.SimCLRProjectionHead(512, 512, 128)
```

Comment:
maybe you can explain the 512,512,128 a bit better unless thats clear for all pros